ir/type.def

```diff
@@ -75,6 +75,7 @@ class Type_Any : Type, ITypeVar {
  protected:
 #emit
     void *operator new(size_t size) { return ::operator new(size); }
```

Comment:
Why do we need new override for these nodes?

Reply:
It was done to make them protected rather than public, so that other code could not directly call `new` and had to call the appropriate `get` method instead.

Reply:
Ah, ok, I missed that `get` methods.